.jenkins/checks.groovy

```diff
@@ -90,7 +92,7 @@ freeStyleJob('cabot.flake8') { job ->
   jobTemplate(job, "Runs tests against Affirm/cabot.git using flake8", 'flake8').call()
   publishers {
     violations {
-      pep8(0, 0, 0, 'build/flake8/flake8.txt')
+      pep8(125, 0, 0, 'build/flake8/flake8.txt')
```

Comment:
Yeah it is, how would you suggest de-duping it? seems a little silly to have a config file just for this...